pkg/gameservers/health.go

```diff
@@ -59,11 +58,13 @@ type HealthController struct {
 }
 
 // NewHealthController returns a HealthController
-func NewHealthController(health healthcheck.Handler,
+func NewHealthController(
+	health healthcheck.Handler,
 	kubeClient kubernetes.Interface,
 	agonesClient versioned.Interface,
 	kubeInformerFactory informers.SharedInformerFactory,
-	agonesInformerFactory externalversions.SharedInformerFactory) *HealthController {
+	agonesInformerFactory externalversions.SharedInformerFactory,
+	waitOnFreePorts bool) *HealthController {
```

Comment:
I didn't see any places in the tests where we set this to true. Are we missing test coverage for that condition?

Reply:
This is covered in https://github.com/googleforgames/agones/blob/35e899d302f5516a9845a13ac699abd410ce7645/pkg/gameservers/health_test.go#L62-L100 (but not at a higher level).